Bot/modules/bot.module.js

```diff
@@ -0,0 +1,250 @@
+'use strict';
+
+const axios = require('axios');
+const UIDGenerator = require('uid-generator');
+const db = require('./db.module');
+const config = require('../../config');
+const { statusChanger, courses } = require('./status.module');
+const ExcelTable = require('./excel.module');
```

Comment:
```suggestion
const db = require('./db.module.js');
const config = require('../../config.js');
const { statusChanger, courses } = require('./status.module.js');
const ExcelTable = require('./excel.module.js');
```

Think hot to avoid `../../`

Reply:
It’s possible to use injections, but it’s not so efficiently

Reply:
It’s more spectacularly